App/UI/MessageHandler.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using App.Players;
+
+namespace App.UI
+{
+    public static class MessageHandler
+    {
+        public static void PrintGreeting()
+        {
+            Console.WriteLine("Welcome to Tic-Tac-Toe!");
+        }
+
+        public static void PrintGameModes()
+        {
+            Console.WriteLine("(1) Play against a friend");
+            // Console.WriteLine("(2) Play against an easy competitor");
+            // Console.WriteLine("(3) Play against a super computer");
+            Console.WriteLine("\n");
+        }
+
+        public static void PrintRequestToChooseGameMode()
+        {
+            Console.Write("Choose from one of the above options: ");
+        }
+
+        public static void PrintRequestToChooseGameModeAfterInvalidInput()
+        {
+            Console.Write("Invalid option. Choose again from options 1-3: ");
+        }
+
+        public static void PrintRequestToInputBoardSize()
+        {
+            Console.Write("Enter board size 3, 4, or 5 (Press enter to default to 3): ");
+        }
+        
+        public static void PrintRequestToInputBoardSizeAfterInvalidInput()
+        {
+            Console.Write("Invalid board size. Enter board size 3, 4, or 5: ");
+        }
+        
+        public static void PrintRequestForPlayerOnesMarker(bool isOpponentComputer)
+        {
+            string title = isOpponentComputer ? "your" : "player one's";
+            Console.Write($"Enter {title} mark (Hit enter to default to \u274C): ");
```

Comment:
maybe extract default marks to variable. I had to look up what symbols \u274C and \u2B55 were